src/internal/operations/test/sharepoint_test.go

```diff
@@ -308,7 +308,8 @@ func (suite *SharePointRestoreNightlyIntgSuite) TestRestore_Run_sharepointDelete
 	var (
 		mb          = evmock.NewBus()
 		counter     = count.New()
-		opts        = control.DefaultOptions()
+		opts        = control.DefaultBackupOptions()
```

Comment:
You're missing a great pun name.  It even sounds like "backups" when you say it out loud.
```suggestion
		backOpts        = control.DefaultBackupOptions()
```